emperor/support_files/js/util.js

```diff
@@ -48,14 +48,13 @@ var k_CHROMABREWER_MAPNAMES = ['Classic QIIME Colors',
 /**
  *
  * Sorting function that deals with alpha and numeric elements
- * 
+ *
```

Comment:
I think I know what happened ...

My editor is configured to automatically remove whitespace every time I save.  
I think there was whitespace before, and my editor deleted it once I saved it.

You sure we want the whitespace?